src/bin/reset_test_instance.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+echo ""
+echo "Check if commit message requires reset"
+git clone --depth=1 https://github.com/greenpeace/"${CONTAINER_PREFIX}"
+GIT_COMMIT_MSG=$(git --git-dir="${CONTAINER_PREFIX}"/.git log --format=%B -n 1 "$CIRCLE_SHA1")
+if [[ $GIT_COMMIT_MSG == *"[RESET]"* && $CONTAINER_PREFIX == *"test-"* ]]; then
+  echo "Reset is required"
+else
+  echo "No Sync required. Exiting."
+  exit 0
+fi
+echo ""
+
+CONTENT_DB_VERSION="0.2.10"
```

Comment:
Once merged I'll add one more script on [defaultcontent](https://github.com/greenpeace/planet4-defaultcontent/blob/f378e0bb193dc9db113cd4538bada38274a16539/.circleci/config.yml#L110-L115) to bump this when we have a new defaultcontent db dump.